README.md

```diff
@@ -147,18 +159,56 @@ See file `Makefile` which currently supports cross compilation. Just run `make`
 
 Also inspect file `bin/md5Sums.txt` after the cross compilation.
 
-
-## Configuration
+#### API keys
 
 Optionally set the `GOOGLE_API_CLIENT_ID` and `GOOGLE_API_CLIENT_SECRET` environment variables to use your own API keys.
 
-## Usage
+#### Scripts
```

Comment:
I am about to create 3 other PRs:
- one for modified README.md
- one for modified platform_packages.md
- one for newly created README-Automation.md
I have also discovered that it is best to create a new branch in my forked repo before submitting any PR, so it should be done.
I think that the current PR should be closed.